src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,66 @@
+@import '../utils/veriables';
```

Comment:
There is a typo in the import statement. It should be `@import '../utils/variables';` instead of `@import '../utils/veriables';`. Ensure the correct path and filename are used.